config.sh

```
@@ -19,6 +19,7 @@ plugins_list_no_jenkinsfile_main_output_file="plugins_no_jenkinsfile.txt"  # Def
 plugins_list_old_java_output_file="reports/plugins_old_java_$current_date.txt"  # Define the output file name.
 plugins_list_old_java_main_output_file="plugins_old_java.txt"
 plugins_list_output_file="plugins.txt"  # Define the output file name.
+plugins_no_jenkinsfile=
```

Shellcheck (0.10.0)

[warning] plugins_no_jenkinsfile appears unused. Verify use (or export if used externally).

(SC2034)